CHANGES.md

```diff
@@ -10,6 +10,10 @@ Change Log
 
 #### next release (8.0.0-alpha.44)
 * Pass `format` trait on `TableColumnTraits` down to `TableAutomaticStylesStratum` for generating legends
+* Added an option for translation override from TerriaMap
+* Help content, trainer bar & help terms can use translation overrides
+* Accepts `backend` options under a new `terria.start()` property, `i18nOptions`
+* [The next improvement]
```

Comment:
Need to move this to the next release? I think we've already release 8.0.0-alpha.44 right?